ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/PaywallViewModel.kt

```diff
@@ -143,7 +149,13 @@ internal class PaywallViewModelImpl(
         }
     }
 
+    @Suppress("TooGenericExceptionCaught")
     private fun Offering.calculateState(): PaywallViewState {
-        return toPaywallViewState(variableDataProvider, mode)
+        if (availablePackages.isEmpty()) {
+            return PaywallViewState.Error("No packages available")
```

Comment:
`RevenueCat` ensures that `Offering` never has empty packages, but this doesn't hurt here 👍🏻